### PR TITLE
Enable language aliases

### PR DESCRIPTION
**Pull request by dennwc**
Update SDK, Go version and enable language aliases.

Addresses https://github.com/src-d/gitbase/issues/728

Signed-off-by: Denys Smirnov <denys@sourced.tech>

---
This change is [Reviewable](https://reviewable.io/reviews/bblfsh/cpp-driver/39)

### CODE REVIEW

**Review by creachadair — APPROVED** [review]
> Reviewed 6 of 6 files at r1.
> *[Reviewable](https://reviewable.io/reviews/bblfsh/cpp-driver/39)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @bzz)

___
*[manifest.toml, line 3 at r1](https://reviewable.io/reviews/bblfsh/cpp-driver/39#-Lff-IB0DGRec-tsBSw9:-Lff-IB0DGRec-tsBSwA:b-mo1zr) ([raw file](https://github.com/bblfsh/cpp-driver/blob/beb357eabc6f93e56ea99d5c50e757b4b9258688/manifest.toml#L3)):*
> ```TOML
> name = "C++"
> language = "cpp"
> aliases = ["C++","C"]
> ```

Completely trivial nit: Space between elements, please (like features).

**Review by dennwc — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/bblfsh/cpp-driver/39)* status: 5 of 6 files reviewed, all discussions resolved (waiting on @bzz and @creachadair)

___
*[manifest.toml, line 3 at r1](https://reviewable.io/reviews/bblfsh/cpp-driver/39#-Lff-IB0DGRec-tsBSw9:-Lff3vJR-yTFwo6jcLoe:b-896fix) ([raw file](https://github.com/bblfsh/cpp-driver/blob/beb357eabc6f93e56ea99d5c50e757b4b9258688/manifest.toml#L3)):*
<details><summary><i>Previously, creachadair (M. J. Fromberger) wrote…</i></summary><blockquote>

Completely trivial nit: Space between elements, please (like features).
</blockquote></details>

Done.

**Review by creachadair — APPROVED** [review]
> Reviewed 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/bblfsh/cpp-driver/39)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @bzz)

File: manifest.toml [hunk 1]
```diff
@@ -1,9 +1,5 @@
 name = "C++"
 language = "cpp"
+aliases = ["C++", "C"]
```
Comment (bzz) [inline manifest.toml]: I would suggest adding `cuda` here as well.

 .cu/.cuh are CUDA [C source files](https://github.com/Haybla/LDPC-CC_Pipeline_Decoder/blob/d814a76df98ebc66a6f94a8595d34e4ab04906ac/src/kernel.cu)/headers for NVCC compiler that will be detected as a separate `Cuda` language.

Comment (dennwc) [reply manifest.toml]: Makes total sense, thanks!

Will also check some other C-like languages from the [list](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml).

Comment (dennwc) [reply manifest.toml]: Done!

**Review by bzz — APPROVED** [review]

**Review by creachadair — APPROVED** [review]
> Reviewed 1 of 1 files at r3, 3 of 3 files at r4.
> *[Reviewable](https://reviewable.io/reviews/bblfsh/cpp-driver/39)* status: :shipit: complete! all files reviewed, all discussions resolved

___
*[manifest.toml, line 3 at r3](https://reviewable.io/reviews/bblfsh/cpp-driver/39#-Lfylp5_74GbARaGGSTX:-Lfylp5_74GbARaGGSTY:b-annnn) ([raw file](https://github.com/bblfsh/cpp-driver/blob/8e4c3a8f07bb1b204db1dc3108a75213f75545d1/manifest.toml#L3)):*
> ```TOML
> name = "C++"
> language = "cpp"
> aliases = ["C++", "C", "CUDA", "OpenCL", "Metal"]
> ```

My immediate reaction was "but we can't index those!" But the syntax is compatible, it's (mainly) only the intrinsics that break type-checking.

**Pull request merged**